openeo/rest/datacube.py

```diff
@@ -628,10 +642,9 @@ def filter_spatial(
             (which will be loaded client-side to get the geometries as GeoJSON construct).
         """
         valid_geojson_types = [
-            "Point", "MultiPoint", "LineString", "MultiLineString",
```

Comment:
In filter_spatial, it's valid to have points and lines, so I don't think this line must be  removed